server/jetstream_cluster_test.go

```diff
@@ -4816,15 +4838,17 @@ var jsClusterImportsTempl = `
 			jetstream: enabled
 			users = [ { user: "rip", pass: "pass" } ]
 			exports [
-				{ service: "$JS.API.>" }
+				{ service: "$JS.API.>", response: stream }
```

Comment:
These updates to the test were needed so that the second message gets delivered.